functions.py

```diff
@@ -0,0 +1,82 @@
+import base64
+import GeodeObjects
+import os
+import pkg_resources
+import werkzeug
+
+def ListAllInputExtensions():
+    """
+    Purpose:
+        Function that returns a list of all input extensions
+    Returns:
+        An ordered list of input file extensions
+    """
+    List = []  # Initiaslizes an empty list
+    ObjectsList = GeodeObjects.ObjectsList() # Dict to loop through
+
+    for Object in ObjectsList.values():
+        values = Object['input']
+        for value in values:
+            list_creators = value.list_creators()
+            for creator in list_creators:  # Loop through
+                if creator not in List:  # If object's name isn't already in the list
+                    List.append(creator)  # Adds the object's name to the listlist
+    List.sort()
```

Comment:
Do you really need the sorting?

Reply:
same question multiple times

Reply:
Since OG doesn't sort the list itself, it allows the output file extensions to be sorted in the UI